src/ext.js

```diff
@@ -1,3 +1,227 @@
+const columnCommonHidden = {
```

Comment:
A lot of complaining from code climate here. That the code look similar in a lot of places is natural, refactoring it would make it less readable IMO. The file is also really long, could be split up if we think it is a good idea.